src/app/dataset-flow/dataset-flow-details/dataset-flow-details.component.ts

```diff
@@ -0,0 +1,123 @@
+import { ChangeDetectionStrategy, Component, OnInit } from "@angular/core";
+import { DatasetFlowByIdResponse, FlowDetailsTabs, ViewMenuData } from "./dataset-flow-details.types";
+import { DatasetViewTypeEnum } from "src/app/dataset-view/dataset-view.interface";
+import { Observable, Subscription, combineLatest, filter, map, shareReplay, switchMap } from "rxjs";
+import {
+    DatasetBasicsFragment,
+    DatasetPermissionsFragment,
+    FlowSummaryDataFragment,
+} from "src/app/api/kamu.graphql.interface";
+import { DatasetInfo } from "src/app/interface/navigation.interface";
+import { Router, ActivatedRoute, NavigationEnd } from "@angular/router";
+import { DatasetService } from "src/app/dataset-view/dataset.service";
+import { DatasetSubscriptionsService } from "src/app/dataset-view/dataset.subscriptions.service";
+import { MaybeUndefined } from "src/app/common/app.types";
+import ProjectLinks from "src/app/project-links";
+import { BaseProcessingComponent } from "src/app/common/base.processing.component";
+import { DatasetFlowsService } from "src/app/dataset-view/additional-components/flows-component/services/dataset-flows.service";
+import { DataHelpers } from "src/app/common/data.helpers";
+import { DatasetFlowTableHelpers } from "src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.helpers";
+
+@Component({
+    selector: "app-dataset-flow-details",
+    templateUrl: "./dataset-flow-details.component.html",
+    styleUrls: ["./dataset-flow-details.component.scss"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DatasetFlowDetailsComponent extends BaseProcessingComponent implements OnInit {
+    public readonly FlowDetailsTabs: typeof FlowDetailsTabs = FlowDetailsTabs;
+    public readonly FLOWS_TYPE = DatasetViewTypeEnum.Flows;
+    public activeTab: FlowDetailsTabs = FlowDetailsTabs.SUMMARY;
```

Comment:
Parts of this class are copies of what we have in main dataset view and block details view.
Consider moving shared things to base class. We've already moved some sub-components for navigation in markup, but we should also move shared things like `datasetBasic$` and `datasetPermission$` - whatever is not specific to flow details - to the new base class.